elephant/statistics.py

```diff
@@ -240,3 +243,83 @@ def lv(v):
     # calculate LV and return result
     # raise error if input is multi-dimensional
     return 3.*np.mean(np.power(np.diff(v)/(v[:-1] + v[1:]), 2))
+
+
+def psth(sts, w, t_start=None, t_stop=None, output='counts', clip=False):
```

Comment:
It is important to get how the binning should work figured out first.  Once it is accepted, then you can submit the PSTH pull request.